### DISCUSSION TITLE
How to manage Scenes of a device with the SceneServer ?

### DISCUSSION THREAD

**Discussion by MatthieuCa** [matter-js/matter.js#1051] [Q&A] [answered]
Hello guys, I will bother you a last time. I am trying to use the ScenesServer because of my project my hardware has the possibility to have multiple scene (more than 200).

I see that there is the ScenesServer cluster on a lot of devices, but I really don't know how it works I understand that there are groups and scenes but how did I use this to make it work?

Is it the good cluster to use or is there another more appropriate ?

I am using matter.js v0.9.3

**Comment by Apollon77** [comment, marked as answer]
The Scenes cluster is still provisional as of Matter 1.3. We had implemented the Matter 1.0 Scenes Server but this was removed and replaced by another (same-ish) cluster which is still provisional.

That means that you most likely will not find any device that really have defined Scenes behavior because noone certified this - and the "old" Scenes Server is as said removed but many devices have it in, but so that's "dead" and there will never be any controller to support it.

We are monitoring the progress on matter sdk and specification side and hope that it becomes usable with Matter 1.4 by end of the year and after this there could show up devices with first Scenes support. Once it is no longer provision we will also add support for it for all clusters that then have support for scenes.
